corehq/apps/app_manager/templates/app_manager/apps_base.html

```diff
@@ -125,18 +133,25 @@
 {% block modals %}
     {{ block.super }}
     {% if app.is_deleted %}
-    <div id="deleted-app-modal" class="modal hide fade">
-        <div class="modal-header"><h3>{% trans 'Oops! This application was deleted.' %}</h3></div>
-        <div class="modal-body">
-            <p>{% trans 'If you want your application back, click Restore.' %}</p>
-        </div>
-        <div class="modal-footer">
-            <a href="{% url "undo_delete_app" domain app.id %}" class="post-link btn btn-primary">
-                {% trans "Restore" %}
-            </a>
-            <a href="#" class="btn" data-dismiss="modal">
-                {% trans "No thanks, get me out of here" %}
-            </a>
+    <!-- This will appear on page load, so don't use any animation (normally controlled by .fade) -->
+    <div id="deleted-app-modal" class="modal">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h3 class="modal-title">{% trans 'Oops! This application was deleted.' %}</h3>
```

Comment:
yeah, i did h3 everywhere, it was what the migration guide happened to have. updated migration guide, will fix the PR.